### DISCUSSION TITLE
implement the .count() methods to AggregationQuery

### DISCUSSION THREAD

**Discussion by mrrobinhood5** [BeanieODM/beanie#720] [Feature Request]
I use the `.count()` method when building FindMany queries. I now need to add an aggregation pipeline to a FindMany query so it makes it an AggregationQuery, but it doesn't have a .count() method. 

example:
```py
base=PlayerBase
search = base.find({}, with_children=True)
# later in the module
search = search.find(RegEx(base.name, name, 'i'))
# later in the module, if a condition is met
search = search.aggregate([{My Pipeline Here...}])
# search gets passed around until finally

count = await search.count()
# this will AttributeError if .aggregate got added somewhere. 
results = await search.to_list()
```

**Comment by akloster** [comment]
You may just need to add another aggregation stage like {"$count":"count"}.

Adding a "count" coroutine to AggregationQuery for convenience could be a valid feature request though.

**Reply by mrrobinhood5** [reply]
Yes, currently that's my work around. It be nice and "pythonic" to have a count co-routine though.
